defaults/main.yml

```diff
@@ -328,14 +328,15 @@ ubuntu1604cis_rsh_required: false
 ubuntu1604cis_ypbind_required: false
 
 # Time Synchronization
-ubuntu1604cis_time_synchronization: chrony
+# ubuntu1604cis_time_synchronization: chrony
 # ubuntu1604cis_time_synchronization: ntp
 
 ubuntu1604cis_time_synchronization_servers:
-  - "0.pool.ntp.org"
-  - "1.pool.ntp.org"
-  - "2.pool.ntp.org"
-  - "3.pool.ntp.org"
+  - 169.254.169.123
```

Comment:
AWS's Time server/service uses a crowdsourced pool of NTP servers that should not be trusted. Please use NIST time servers from here:
https://tf.nist.gov/tf-cgi/servers.cgi

Reply:
I think we should alter the DHCP option set for the VPC rather than mandate it here. This variable only supports configuration of either ntp or chrony daemons. `timesyncd` is usually configured to prefer servers from DHCP advertisement

Reply:
I have no preferences about how we enforce what time servers are looked up - whether we do it through VPC config or local config - it is up to Platform to decide the design. Aside from security concern about using AWS' time service, we should have a list of time servers in case the primary fails.